core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2729,7 +2920,381 @@ components:
+        count:
+          type: integer
+          format: int32
+          minimum: 0
+          maximum: 10000
+          description: An integer between `0` and `10000`, giving the total count of transactions in the returned response
+        max_ledger_state_version:
+          type: integer
+          format: int64
+          minimum: 1
+          maximum: 100000000000000
+          description: An integer between `1` and `10^13`, giving the maximum state version currently committed on this node's ledger.
+        committed_transaction_basic_outcomes:
+          description: A committed transaction basic outcomes list starting from the `from_state_version` (inclusive).
+          type: array
+          items:
+            $ref: "#/components/schemas/LtsCommittedTransactionFungibleOutcome"
+    LtsCommittedTransactionStatus:
+      type: string
+      enum:
+        - Succeeded
+        - Failed
+      description: The status of the transaction
+    LtsCommittedTransactionFungibleOutcome:
+      type: object
+      required:
+        - state_version
+        - accumulator_hash
+        - status
+        - fungible_outcome
+      properties:
+        state_version:
+          type: integer
+          format: int64
+          minimum: 1
+          maximum: 100000000000000
+          description: An integer between `1` and `10^13`, giving the resultant state version after the transaction has been committed
+        accumulator_hash:
+          type: string
+          minLength: 64
+          maxLength: 64
+          description: |
+            The hex-encoded transaction accumulator hash. This hash captures the order of all transactions on ledger.
+            This hash is `ACC_{N+1} = Blake2b-256(CONCAT(ACC_N, LEDGER_HASH_{N}))`, starting with `ACC_0 = 000..000` the pre-genesis accumulator.
+        status:
+          $ref: "#/components/schemas/LtsCommittedTransactionStatus"
+        fungible_outcome:
+          description: A list containing all account fungible balance updates
+          type: array
+          items:
+            $ref: "#/components/schemas/LtsAccountFungibleBalanceChanges"
+    LtsAccountFungibleBalanceChanges:
+      type: object
+      required:
+        - account_address
+        - fungible_resource_balance_changes
+      properties:
+        account_address:
+          type: string
+          description: The Bech32m-encoded human readable version of the account's address
+        fungible_resource_balance_changes:
+          type: array
+          items:
+            $ref: "#/components/schemas/LtsFungibleResourceBalanceChange"
+    LtsFungibleResourceBalanceChange:
+      type: object
+      required:
+        - fungible_resource_address
+        - balance_change
+      properties:
+        fungible_resource_address:
+          type: string
+          description: The Bech32m-encoded human readable version of the fungible resource's global address
+        balance_change:
```

Comment:
We also need to include fee payment information. I wonder if we should have an optional field here `fee_payment` (which is only included if the amount is XRD)?

And include in the description for `balance_change` that it _includes_ the fee payment?